### ISSUE TITLE
New event ID for same consumer group and unchanged status

### ISSUE THREAD

**Issue by zako** [linkedin/Burrow#314]
I upgraded to Burrow version 1.0.0 and observed that if a consumer group is STOPPED status then a new event ID is generated each time the groups are evaluated. This is problematic because in the older version of Burrow the event IDs were consistent across evaluations and changed only when the status changed. I use event ID in a custom HTTP server to track status based on event ID (from ERR -> OK) and now the status for a particular consumer group cannot be tracked because the event IDs are always unique. Is this expected?

I narrowed down the code responsible for this unexpected behavior in notifier/coordinator.go:
```
func (nc *Coordinator) processConsumerList(cluster string, replyChan chan interface{}) {
        ...
	if _, ok := nc.clusters[cluster]; ok {
		nc.clusters[cluster].Lock.Lock()
		consumerMap := make(map[string]struct{})
		for _, group := range consumerList {
			consumerMap[group] = struct{}{}
			// existing consumerGroup entry is overwritten
			nc.clusters[cluster].Groups[group] = &consumerGroup{
				LastNotify: make(map[string]time.Time),
				LastEval:   time.Now().Add(-time.Duration(rand.Int63n(nc.minInterval*1000)) * time.Millisecond),
			}
		}
```
I added custom logging before and after the method above is executed. Here is the output:
```
{"level":"info","ts":1513884872.6307108,"msg":"before processConsumerList group:dev_test id:68a64412-78e5-4d74-9cd9-0227072e20fa start:2017-12-21 13:34:20.816610438 -0600 CST m=+229.187107501","type":"coordinator","name":"notifier"}
{"level":"info","ts":1513884872.6307294,"msg":"after processConsumerList group:dev_test id: start:0001-01-01 00:00:00 +0000 UTC","type":"coordinator","name":"notifier"}
```
Disclaimer, I am new to Go and could be misinterpreting the code.

**Comment by toddpalino** [comment]
This does appear to be a real issue. The current tests don't catch it, as it requires both an existing incident and then a refresh of the groups (we test them separately). It should be solvable by checking to see if the group exists before blindly overwriting it, and just doing nothing if it already exists.

**Comment by rconn01** [comment]
I actually ran into this issue as well, didn't see that an issue already existed for it. I proposed a fix as part of #334

**Comment by zako** [comment]
Good find, @rconn01. I was about to update this issue with the same behavior. I noticed that alerts were never being closed because the start time was always zero. Therefore, my custom http endpoint only alerted when there was an error but never resolved it when switched to OK state.

I will update Burrow to get the fix.

**Comment by toddpalino** [comment]
Closed by #334
